docs/REST.md

```diff
@@ -202,6 +203,37 @@ Content-Type: application/json
     }
 }
 ```
+
+## PATCH `/api/model/instock`
+
+Updates an unserialized model's total and instock values after adding a new one
+
+### Parameters
+
+* `modelAddress`: Address of the model to increment
```

Comment:
The only time I use address is inside an actual model or item object.  Everywhere else I specify, obvious or not.